ElectronClient/gui/StatusScreen.jsx

```diff
@@ -114,7 +115,30 @@ class StatusScreenComponent extends React.Component {
 				sectionsHtml.push(renderSectionHtml(i, section));
 			}
 
-			return <div>{sectionsHtml}</div>;
+			const onClick = async () => {
+				for (let i = 0; i < report.length; i++) {
+					let section = report[i];
+					if (!section.body.length) continue;
+					for (let n in section.body) {
+						let item = section[n];
+						if (typeof item === 'object') {
+							if (item.canRetry()) {
+								await item.retryHandler();
+							}
+						}
+					}
+				}
+				this.resfreshScreen();
+			};
+
+			const retryAllLink = <a href='#' style={retryAllStyle} onClick={onClick}>{_('Retry All')}</a>;
+
+			return (
+				<div>
+					{retryAllLink}
+					<br />
+					{sectionsHtml}
+				</div>);
```

Comment:
The feature request was for a Retry All button when items cannot be decrypted. But with this code, you'll add a Retry All button that will retry everything, including decryption, downloading resources again, etc.

Also in general it's best not to put business logic like that in the view. All this should be in services/report.js. Maybe have a look at this file and see how the sections are built, then add your Retry All buttons (there should be two of them) at the right place.

The advantage of doing it that way is that it will work in the mobile app too.